e2e/pages/widgets/account-settings/profile-form.js

```diff
@@ -0,0 +1,61 @@
+import { Select } from "../shared/select";
+import { Input } from "../shared/input";
+
+export class UserProfileForm {
+  firstNameInput = Input.byLabel("First name");
+  lastNameInput = Input.byLabel("Last name");
+  emailInput = Input.byLabel("Email");
+  // TODO: make it more specific
+  languageSelect = Select.byTestId("select-button");
+  submitButton = () => cy.button("Update");
```

Comment:
Could we add `Button` wrapper in `shared`, too?

Reply:
This would also allow us to use it like `this.submitButton` and not `this.submitButton()`